go/vt/vtctl/workflow/utils.go

```diff
@@ -851,34 +851,41 @@ func getTenantClause(vrOptions *vtctldatapb.WorkflowOptions,
 }
 
 func changeKeyspaceRouting(ctx context.Context, ts *topo.Server, tabletTypes []topodatapb.TabletType,
-	sourceKeyspace string, targetKeyspace string) error {
+	sourceKeyspace, targetKeyspace, reason string) error {
 	routes := make(map[string]string)
 	for _, tabletType := range tabletTypes {
 		suffix := getTabletTypeSuffix(tabletType)
 		routes[sourceKeyspace+suffix] = targetKeyspace
 	}
-	if err := updateKeyspaceRoutingRule(ctx, ts, routes); err != nil {
+	if err := updateKeyspaceRoutingRule(ctx, ts, sourceKeyspace, reason, routes); err != nil {
 		return err
 	}
 	return ts.RebuildSrvVSchema(ctx, nil)
 }
 
 // updateKeyspaceRoutingRule updates the keyspace routing rule for the (effective) source keyspace to the target keyspace.
-func updateKeyspaceRoutingRule(ctx context.Context, ts *topo.Server, routes map[string]string) error {
-	rules, err := topotools.GetKeyspaceRoutingRules(ctx, ts)
+func updateKeyspaceRoutingRule(ctx context.Context, ts *topo.Server, sourceKeyspace, reason string, routes map[string]string) error {
```

Comment:
All other funcs have been named with `Rules` and this one with `Rule`. That should probably change to be consistent.

Reply:
Done